src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -568,7 +568,7 @@ public void addPayloadType(PayloadType payloadType)
     public long getLastActivity()
     {
         PacketIOActivity packetIOActivity
-                = this.transceiver.getPacketIOActivity();
+                = this.dtlsTransport.getPacketIOActivity();
```

Comment:
i'm wondering if there's actually a use-case to distinguish last ICE activity from last media activity, in which case we should keep the `PacketIOActivity` instance in the `Transceiver` and just update it from here, rather than replacing that altogether.  if we _do_ go this route, we need an accompanying PR in JMT to remove `PacketIOActivity` from `Transceiver` so it's not accidentally used, I think.

I think I'm in favor of keeping this in `Transceiver` and just updating that instance from here, as then we can more easily expand the functionality here (separately tracking ICE and RTP activity, maybe even RTP/RTCP/audio/video activity, etc...we've got options for a lot more depth).

Reply:
> i'm wondering if there's actually a use-case to distinguish last ICE activity from last media activity, 

@bbaldino  There is one use-case I used. During hold (no media sent between client and jvb) endpoint is considered as disconnected, despite SCTP and ICE packets being sent/received.

> we need an accompanying PR in JMT to remove PacketIOActivity from Transceiver so it's not accidentally used, I think.

That's sounds reasonable.

> I think I'm in favor of keeping this in Transceiver and just updating that instance from here,

Have no objection to that. The only thing I wanted to fix actually is to properly handle case when no media is transmitted, but endpoint is actually connected to jvb. Today such endpoint is considered as having connectivity issues.

Reply:
> Have no objection to that. The only thing I wanted to fix actually is to properly handle case when no media is transmitted, but endpoint is actually connected to jvb. Today such endpoint is considered as having connectivity issues.

Yep, agree.  Let me take a quick look at updating `PacketIOActivity` in JMT.  I'll put something up and let me know what you think.

Reply:
What do you think of [this?](https://github.com/jitsi/jitsi-media-transform/pull/148)  You could then update `lastIceActivityTimestamp` here and query `latestOverallActivity` which would include ICE activity.

Reply:
@mstyura https://github.com/jitsi/jitsi-media-transform/pull/148 just got merged...want to re-do this on top of that?

Reply:
@bbaldino thanks, I could redo but only tomorrow. If you have time and wish to do it yourself feel free to do so :)

Reply:
@bbaldino I've done change for ICE, but I think it also useful to have last SCTP send/receive timestamp. I could add it as well, but I prefer followup PR, because it will require PR to JMT.